lib/commits.js

```diff
@@ -92,6 +92,68 @@ const findCommitsWithAssociatedPullRequestsQuery = /* GraphQL */ `
   }
 `
 
+const diffCommitsWithAssociatedPullRequestsQuery = /* GraphQL */ `
+  query diffCommitsWithAssociatedPullRequests(
+    $name: String!
+    $owner: String!
+    $baseRef: String!
+    $targetCommitish: String!
+    $withPullRequestBody: Boolean!
+    $withPullRequestURL: Boolean!
+    $withBaseRefName: Boolean!
+    $withHeadRefName: Boolean!
+  ) {
+    repository(name: $name, owner: $owner) {
+      ref(qualifiedName: $baseRef) {
+        compare(headRef: $targetCommitish) {
```

Comment:
the goal here is to get something like the result of this rest endpoint:
https://docs.github.com/en/rest/commits/commits?apiVersion=2022-11-28#compare-two-commits